src/standard_models.jl

```diff
@@ -50,6 +57,7 @@ DOI:10.1103/PhysRevLett.77.3865
 function model_PBE(lattice::AbstractMatrix, atoms::Vector, positions::Vector; kwargs...)
     model_DFT(lattice, atoms, positions, [:gga_x_pbe, :gga_c_pbe]; kwargs...)
 end
+model_PBE(system::AbstractSystem; kwargs...) = call_with_system(model_PBE, system; kwargs...)
```

Comment:
You can do all of these with an eval loop: `for fun in (:a, :b) @ eval $fun() ...`